docs/prometheus.md

```diff
@@ -0,0 +1,68 @@
+# Collecting Prometheus Data
+
+How To:
+* [What is it](#what-is-it)
+* [Configuring](#configuring)
+
+# What is it
+
+The ability to capture prometheus data during a benchmark run was added to the [Benchmark-wrapper](https://github.com/cloud-bulldozer/benchmark-wrapper). 
+To enable this functionality and pass prometheus data to an Elasticsearch server the workload CR must be appropriately configured.
+
+
+# Configuring
+
+To enable this functionality a few variables must be set in the workload CR file.
+
+```
+prometheus:
+  server: the elasticsearch server to upload to
+  port: the elasticsearch server port
+  prom_url: the prometheus URL
+  prom_token: a valid access token for prometheus
+  parallel: enable parallel uploads to elasticsearch
```

Comment:
Can you mention that the "parallel" argument is also available in the metadata collection?

Reply:
I thought about that but really didnt know where it would be best to put it. We should probably just duplicate this doc for the ES section. I'll add it in a few.